Automattic/jetpack
# Jetpack AI: Add feedback control to ask for feedback on the block inspector

Fixes #34471.

## Proposed changes:

* Create the FeedbackControl component as a place to ask for feedback
   * The actual feedback format is an external link that is pointing to the general Jetpack Feedback form, but that can be changed to any other location if we find a better location
   * We can expand this in the future to collect quantitative feedback about the overal results of the assistant or any other format that we find suitable
* Place the FeedbackControl component on the AI Assistant block inspector sidebar
* Update the block description to drop repeated line about feedback

| Before | After |
| --- | --- |
| <img width="250" alt="Screenshot 2023-12-07 at 16 38 54" src="https://github.com/Automattic/jetpack/assets/6760046/42edce1d-e4da-48d6-8ba3-62d4bd2a7f8e"> | <img width="250" alt="Screenshot 2023-12-07 at 16 33 08" src="https://github.com/Automattic/jetpack/assets/6760046/066c4ac1-2770-4861-b7e7-9a2e90a1140a"> |

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Run this branch on your test site or launch a new JN site with it
* Go to the block editor
* Insert an AI Assistant block
* Confirm that you see the "Feeback" section on the block inspector
* Confirm that the link is pointing to the `https://jetpack.com/redirect/?source=jetpack-ai-feedback` redirect
* Confirm that the link opens on a new tab
* Confirm that you are redirected to `https://jetpack.com/submit-feedback/`
